### DISCUSSION TITLE
微信机器人，监听不到图片、视频、语音信息

### DISCUSSION THREAD

**Discussion by Hly-Maoxiong** [nonebot/discussions#325] [问答]
在使用微信机器人时，使用ntchat-client微信登录成功，并且与nonebot连接成功，可以监听到文本信息，可是却监听不到图片、文件、语音、视频信息。
![mmexport1677137877551](https://user-images.githubusercontent.com/126145911/220845308-582511c4-d4fd-403a-bc1b-3919989c2f4d.jpg)
![mmexport1677137851311](https://user-images.githubusercontent.com/126145911/220845374-9162cc60-6da6-48c8-b21f-73ffff3656cf.jpg)

**Comment by yanyongyu** [comment]
@JustUndertaker

**Comment by Saya054** [comment]
你可以用[JustUndertaker]大佬的ComWechatBotClient项目，那个项目我试了可以监听到图片、声音、视频的事件
